### ISSUE TITLE
chracters in jail

### ISSUE THREAD

**Issue by knightmere** [LegionDark/Issues#1718]
**Date & Time**: not sure on the date but current is 12/11/17

**FFXI Client Version (use `/ver`)**: 3017110

**Server's Expected Client Version matches, yes/no? (use `!ecv`)**: 

**Character Name**: knightmere,savior,darkseyes

**Nation**: 
windy

**Job(level)/Sub Job(level)**: knightmere  pld/war    savior rdm/blm     darseyes  thf/nin

**NPC or Monster or item Name**: 

**Zone name**: 

**Coordinates (use `!where`)**: 

**ffxiah.com link (for items issues only)**: 

**Multi-boxing? (multiple clients on same connection)**: 

**Steps To Reproduce / any other info**:

**Comment by R3P0FFXI** [comment]
@knightmere 

This happens from using tools or addons that try to strip off status effects that you normally cannot.

Each time that happens the code will re-jail. Even if you are already in jail. Please disable any such tools and remove them and any windower or ashita scripts or macros that would attempt to automatically remove such effects (like weakness) so that you don't auto-jail yourself again, and I will release you at my earliest convenience. And hopefully I never see you jail yourself again.

tl;dr
 be with you when I get a chance

**Comment by knightmere** [comment]
k

**Comment by EpicTaru** [comment]
Did disable the Debuff or Cancel addon, @knightmere?  You'll end up right back in jail if you did not.

**Comment by knightmere** [comment]
yes

**Comment by EpicTaru** [comment]
Log your charas back on please.  They need to be online in order to be released.

**Comment by EpicTaru** [comment]
Released 12/11/17.
